src/engine/mod.rs

```diff
@@ -139,17 +141,20 @@ impl Engine {
     /// ```no_run
     /// use fyrox::engine::Engine;
     /// use fyrox::window::WindowBuilder;
+    /// use fyrox::resource_manager::ResourceManagerBuilder;
     /// use fyrox::event_loop::EventLoop;
     ///
     /// let evt = EventLoop::new();
+    /// let resource_manager_builder = WindowBuilder::new()
     /// let window_builder = WindowBuilder::new()
     ///     .with_title("Test")
     ///     .with_fullscreen(None);
-    /// let mut engine: Engine = Engine::new(window_builder, &evt, true).unwrap();
+    /// let mut engine: Engine = Engine::new(window_builder, resource_manager_builder, &evt, true).unwrap();
     /// ```
     #[inline]
     pub fn new(
         window_builder: WindowBuilder,
+        resource_manager_builder: ResourceManagerBuilder,
```

Comment:
This should accept `ResourceManager` itself, not a builder. First parameter (`window_builder`) is passed as builder because it is used by glutin's API.

Reply:
done